Doc/whatsnew/3.13.rst

```diff
@@ -157,6 +157,10 @@ Removed
   use the :mod:`subprocess` module instead.
   (Contributed by Victor Stinner in :gh:`104773`.)
 
+* :pep:`594`: Remove the :mod:`!ossaudiodev` module, deprecated in Python 3.11:
+  use the `pygame project <https://www.pygame.org/>`_ for audio playback.
```

Comment:
Shall we also/instead mention the pygame-ce fork? https://pyga.me

More info: https://www.reddit.com/r/pygame/comments/1112q10/pygame_community_edition_announcement/

Reply:
I would prefer to only link to pygame for now. It's unclear to me where the community goes, but so far, "pygame" remains the reference project. I hope that the two communities will merge again. If not, we can update the doc later. The problem is that the Python documentation is not updated frequently.